pkg/cableengine/syncer/syncer.go

```diff
@@ -70,41 +70,41 @@ func NewGatewaySyncer(engine cableengine.Engine, client v1typed.GatewayInterface
 	}
 }
 
-func (s *GatewaySyncer) Run(stopCh <-chan struct{}) {
+func (gws *GatewaySyncer) Run(stopCh <-chan struct{}) {
```

Comment:
Usually 1 or 2 letters suffices as outlined [here](https://github.com/golang/go/wiki/CodeReviewComments#receiver-names).

```suggestion
func (gs *GatewaySyncer) Run(stopCh <-chan struct{}) {
```